proxy-config/local-frontend.js

```diff
@@ -9,9 +9,10 @@ module.exports = {
         '/openshift/': { host: 'PORTAL_BACKEND_MARKER' },
         '/hybrid/': { host: 'PORTAL_BACKEND_MARKER' },
         '/api/': { host: 'PORTAL_BACKEND_MARKER' },
-        '/migrations/': { host: 'PORTAL_BACKEND_MARKER'},
-        '/ansible/': { host: 'PORTAL_BACKEND_MARKER'},
-        '/subscriptions/': { host: 'PORTAL_BACKEND_MARKER'},
-        '/': { host: 'https://localhost:8002' }
+        '/migrations/': { host: 'PORTAL_BACKEND_MARKER' },
+        '/ansible/': { host: 'PORTAL_BACKEND_MARKER' },
+        '/subscriptions/': { host: 'PORTAL_BACKEND_MARKER' },
+        '/': { host: 'https://localhost:8002' },
+        '/config': { host: 'https://ci.cloud.redhat.com' }
```

Comment:
You can actually set this to `PORTAL_BACKEND_MARKER` as well.

Reply:
@karelhala changed